### PR TITLE
nickname ui test fix

### PR DESCRIPTION
**Pull request by g-harel**

### CODE REVIEW

File: test/androidTest/java/org/thoughtcrime/securesms/espresso/ViewActions.java [hunk 1]
```diff
@@ -69,4 +69,45 @@ public void perform(UiController uiController, View view) {
...
+
+            @Override
+            public String getDescription() {
+                return "clicking on parent view";
+            }
+
+            @Override
+            public void perform(UiController uiController, View view) {
+                for (int i = 0; i < levels; i++) {
+                    view = (View)view.getParent();
```
Comment (Radu-Raicea) [inline test/androidTest/java/org/thoughtcrime/securesms/espresso/ViewActions.java:88]: Space after type casting.

**Review by Radu-Raicea — CHANGES_REQUESTED** [review]

File: test/androidTest/java/org/thoughtcrime/securesms/espresso/ViewActions.java (hunk 1 above)
Comment (g-harel) [reply test/androidTest/java/org/thoughtcrime/securesms/espresso/ViewActions.java:88]: I've looked in the codebase and that doesn't seem to be the pattern

Comment (Radu-Raicea) [reply test/androidTest/java/org/thoughtcrime/securesms/espresso/ViewActions.java:88]: Ah, it seems that Signal does not respect the official [Oracle Java Style Guide](http://www.oracle.com/technetwork/java/codeconventions-150003.pdf).

Then it's okay.

**Review by Radu-Raicea — APPROVED** [review]

**Review by Dan-Yoo — APPROVED** [review]
looks good

**Pull request merged**